{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -65,10 +65,10 @@ commands =
     {%- endif %}
 {%- else %}
     {%- if cookiecutter.test_matrix_separate_coverage|lower == 'yes' %}
-    nocov: {posargs:nosetests -v tests}
```

Comment:
I fact ``nosetes`` read the ``setup.cfg`` if it is started in the project's directory.
See: http://nose.readthedocs.io/en/latest/usage.html#configuration
Is the current working directory set during ``tox`` task?